vercel/turbo
# feat(Turborepo): Use package terminology (and do version bumps)

### Description

 - Use package terminology instead of workspace
 - bump versions in preparation for a publish
 - script version bump (not wired to the publish step yet)

### Testing Instructions

 - add explicit imports in the tests for everything we expect to be exporting
 - add a very mild test for package manager detection

Closes TURBO-1785
